alexandria/core/models.py

```diff
@@ -268,12 +269,18 @@ def set_content_vector(self):
             config=config,
             weight="B",
         )
+        self.save()
 
     def save(
         self, force_insert=False, force_update=False, using=None, update_fields=None
     ):
         self.set_checksum()
```

Comment:
Suggest moving the `set_checksum()` code into a Celery task as well. That way, it can (theoretically) run in parallel to the content vector calculation

Reply:
Done